index.html

```diff
@@ -45,5 +45,118 @@ <h2>LET’S CONNECT</h2>
+                </p>
+                <ul class="tag">
+                    <li>HTML</li>
+                    <li>CSS</li>
+                    <li>JavaScript</li>
+                </ul>
+                <button type="button">See project</button>
+            </article>
+            <article class="card">
+                <img src="./assets/Snapshoot3.svg" alt="Snapshoot">
+                <h2>Tonic</h2>
+                <ul class="primary-text">
+                    <li>CANOPY</li>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                </ul>
+                <p class="paragraph">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="tag">
+                    <li>HTML</li>
+                    <li>CSS</li>
+                    <li>JavaScript</li>
+                </ul>
+                <button type="button">See project</button>
+            </article>
+            <article class="card">
+                <img src="./assets/Snapshoot4.svg" alt="Snapshoot">
+                <h2>Multi-Post<br>Stories</h2>
+                <ul class="primary-text">
+                    <li>CANOPY</li>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                </ul>
+                <p class="paragraph">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="tag">
+                    <li>HTML</li>
+                    <li>CSS</li>
+                    <li>JavaScript</li>
+                </ul>
+                <button type="button">See project</button>
+            </article>
+        </section>
+        <section id="about">
+            <h2 class="header">About Myself</h2>
+            <p class="paragraph">
+                Hello I’m a software developer! I can help you build a product, feature or website. 
+                Look through some of my work and experience! If you like what you see and have a project 
+                you need coded, don’t hesitate to contact me.
+            </p>
+            <h2 class="header-2">LET’S CONNECT</h2>
+            <ul class="social-icon social-icon-2">
+                <li><a href="#"><img src="./assets/github.svg" alt="Twitter"></a></li>
+                <li><a href="#"><img src="./assets/Linkedin.svg" alt="LinkedIn"></a></li>
+                <li><a href="#"><img src="./assets/angellist.svg" alt="Medium"></a></li>
+                <li><a href="#"><img src="./assets/twt.svg" alt="GitHub"></a></li>
+                <li><a href="#"><img src="./assets/meduim.svg" alt="AngelList"></a></li>
+            </ul>
+            <button type="button" id="reume">Get my resume</button>
+            <ul id="tech">
+                <li>Languages</li>
+                <li>Frameworks</li>
+                <li>Skills</li>
```

Comment:
kindly implement the languages section of your code to be the one given in the design 😄 

Design given :
![image](https://github.com/sagieramos/Portfolio/assets/20537712/20dceb5b-a0cf-49b9-a1bb-867bc8d116f3)


Your design :
![image](https://github.com/sagieramos/Portfolio/assets/20537712/2801b14c-27f0-4c7a-ba05-3f94756d0ca5)